tests/skeleton.py

```diff
@@ -176,11 +176,11 @@ def put(self, data, url=None, envelope_data=True, role=None, user=None,
             version=version, content_type=content_type, headers=headers)
 
     def init_dep(self):
-        dep = models.Departement()
-        dep.nom = "Mayenne"
-        dep.numero = "53"
+        if models.Departement.query.filter_by(numero="53").all() != []:
```

Comment:
Can be replaced with `if models.Department.query.filter_by(numero="53").count():`

Reply:
To check if a list is empty, linters (pylint, pyflake, ...) advise not to compare against an empty list but rather to use the "booleanness" of the list.

Instead of `if mylist != []:`, it is preferable to use `if mylist:`